### PR TITLE
Allow custom data path for graphql pull replicatoin

### PR DESCRIPTION
**Pull request by angelnar87**
#2606
This PR allows a developer to specify a custom response data path for syncGraphQL

- [x] A new feature

TODO
- [ ] tests
- [ ] docs
- [ ] typings
- [ ] changelog

### CODE REVIEW

File: src/plugins/replication-graphql/index.ts [hunk 1]
```diff
@@ -198,6 +199,17 @@ export class RxGraphQLReplicationState {
         return false;
     }
 
+    private getIn(data: any, path: string[] = []): any[] {
```
Comment (pubkey) [inline src/plugins/replication-graphql/index.ts:202]: We should use `object-path` instead. This is already used many times in the project.
https://github.com/pubkey/rxdb/search?q=object-path

**Comment by pubkey** [comment]
Thanks for the PR. I added a comment.

File: src/plugins/replication-graphql/index.ts (hunk 1 above)
Comment (angelnar87) [reply src/plugins/replication-graphql/index.ts:202]: Sounds good to me. I'll make the change

**Comment by pubkey** [comment]
@angelnar87  any update on this?

**Pull request closed without merging**